src/main/java/io/cryostat/net/web/http/generic/RequestLoggingHandler.java

```diff
@@ -114,6 +123,74 @@ public void handle(RoutingContext ctx) {
         this.delegate.handle(ctx);
     }
 
+    // this is ripped from the LoggerHandler's `DEFAULT` format, but with the addition of the
```

Comment:
Just had a thought, I think it may be nice to have color coding in the terminal for better readability. For example just using something like https://stackoverflow.com/questions/5762491/how-to-print-color-in-console-using-system-out-println here in your Logger formatter implementation. Thoughts? I'm not sure how it would work for other Logger implementations currently, but it could be something for me to look into if it's not straight-forward.

Reply:
Sure, that could be a nice enhancement. It would probably go along with refactoring our `Logger` stuff entirely, though. Injecting the `Logger` instance with Dagger means that everything uses the same single instance, which is a tiny performance gain but comes at the cost of not being able to tell which class is logging the message easily. More commonly, a class gets a Logger instance like ex. `private static final Logger LOG = LoggerFactory.getLogger(MyClass.class)`, and then the `LOG` instance can use `MyClass` as a prefix for the messages it logs, and the log config can even set up different default log levels on a per-class basis etc. If you're interested in taking on that work then adding in colourized output for the class name prefix and a timestamp might make it worthwhile.

Just note however that those colour escape codes are generally only meaningful and useful when the output is being viewed directly in a terminal and not when the logs are being written to a file. There are other tools that an end user could use to strip the colour codes out of the logs but it might be more annoying to deal with. For that reason I would suggest making the colour-coding an opt-in feature by env var or system property so we can just enable it during `devserver.sh`/`run.sh`/`smoketest.sh` runs and leave it off by default elsewhere.